spectrum/handlers/silent.py

```diff
@@ -0,0 +1,21 @@
+import logging
+from concurrent.futures import ThreadPoolExecutor
+
+
+class DisableLogger():
+    def __init__(self, level):
+        self.level = level
+
+    def __enter__(self):
+        logging.disable(self.level)
+
+    def __exit__(self, *args, **kwargs):
+        logging.disable(logging.NOTSET)
+
+
+class SilentExecutor(ThreadPoolExecutor):
```

Comment:
this executor class is needed for when Spectrum is closed while `runserver` is still running